MergerCli/IProcess.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using MergerCli.Utils;
 using MergerLogic.DataTypes;
-using MergerLogic.ImageProcessing;
 
 namespace MergerCli
 {
     internal interface IProcess
     {
-        void Start(TileFormat targetFormat, IData baseData, IData newData, BatchStatusManager batchStatusManager);
+        void Start(IData baseData, IData newData, BatchStatusManager batchStatusManager);
```

Comment:
should there be some default format - by configuration or design? so this param not be requested? not familiar with the correct design for this scenario

Reply:
It is read from the `outputFormat` env under `TILE` in the `appsettings.json` file, and inserted as part of the new `TileFormatStrategy`. So it isn't needed any more as a parameter for the `Start` function.